addons/marker_flags/CfgVehicles.hpp

```diff
@@ -0,0 +1,51 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class ADDON {
+                    displayName = CSTRING(ActionPlace);
+                    condition = QUOTE(_player call FUNC(canPlace));
+                    insertChildren = QUOTE(_this call FUNC(addActions));
+                };
+            };
+        };
+    };
+
+    class FlagMarker_01_F;
+    class GVAR(white): FlagMarker_01_F {
+        scope = 2;
+        scopeCurator = 2;
+        author = ECSTRING(common,ACETeam);
+        displayName = CSTRING(white);
+        hiddenSelectionTextures[] = {"#(argb,8,8,3)color(1,1,1,1,co)"};
```

Comment:
```suggestion
        hiddenSelectionsTextures[] = {"#(argb,8,8,3)color(1,1,1,1,co)"};
```
Same for the rest :)